### DISCUSSION TITLE
Peptide back to MS2 scan

### DISCUSSION THREAD

**Discussion by PSobrevalsAlcaraz** [vdemichev/DiaNN#211] [Q&A]
Hello Vadim,

I have been doing several analyses with DiaNN, and now I would like to dive in a little bit more and be able to see the peptides in its own scan. I have seen there's a column in the Report.tsv file named MS2.Scan, but that doesn't seem to correspond with the raw file ms2 scan... We have a peptide with a PrecursorMZ of 616.85 which is identified in an MS2 scan with window of mz 720-730.

I was wondering if DiaNN creates its own MS2 scan numbers? Or how could we properly match the peptide back to the raw file scan?

Thank you so much for your time and for developing such a nice and useful tool :)

Paula

**Comment by vdemichev** [comment]
Hi Paula,

DIA-NN scan numbers start with 0. And MS2 scans are numbered one after another, the numbering for MS1 ones is separate. That is the first MS2 scan has number 0, and the first MS1 scan also has number 0.

As for visualisation, please try the --vis command in DIA-NN.

Best,
Vadim

**Reply by PSobrevalsAlcaraz** [reply]
Hi Vadim,

Thank you so much for your help, and quick reply!! Really useful :))

Have a nice day :)

Paula

**Reply by PSobrevalsAlcaraz** [reply]
Hi @vdemichev ,

I have analysed 21 files, and the MS2.Scan reported in Report.tsv of a raw file are way higher than the actual number of scans: The raw file 11939 scans and in the Report.tsv file they start at scan 27498 until 53218.
Does it mean that DiaNN renames all MS2 concatenating the raw files? 
I mean, if for example I had raw_file1.raw, raw_file2.raw and raw_file3.raw: would the ms2 scans be renamed as: 
A)   
<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/MAXQUA~1/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/MAXQUA~1/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">
<style>

</style>
</head>

<body link="#0563C1" vlink="#954F72">



Raw File | Scan Number
-- | --
raw_file1.raw | **MS2.Scan0**
raw_file1.raw | MS2.Scan1
raw_file1.raw | MS2.Scan2
raw_file1.raw | MS2.Scan3
raw_file2.raw | **MS2.Scan4**
raw_file2.raw | MS2.Scan5
raw_file2.raw | MS2.Scan6
raw_file3.raw | **MS2.Scan7**
raw_file3.raw | MS2.Scan8
raw_file3.raw | MS2.Scan9



</body>

</html>


or

B)

<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/MAXQUA~1/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/MAXQUA~1/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">
<style>

</style>
</head>

<body link="#0563C1" vlink="#954F72">



Raw File | Scan Number
-- | --
raw_file1.raw | **MS2.Scan0**
raw_file1.raw | MS2.Scan1
raw_file1.raw | MS2.Scan2
raw_file1.raw | MS2.Scan3
raw_file2.raw | **MS2.Scan0**
raw_file2.raw | MS2.Scan1
raw_file2.raw | MS2.Scan2
raw_file3.raw | **MS2.Scan0**
raw_file3.raw | MS2.Scan1
raw_file3.raw | MS2.Scan2



</body>

</html>


Thank you so much again for your time!

Best,
Paula

**Reply by vdemichev** [reply]
Dear Paula,

Scan numbers reported by DIA-NN are run-specific and don't depend on the other files. What is (i) gradient (ii) cycle time and (iii) number of DIA windows for the run in question?

Best,
Vadim

**Reply by PSobrevalsAlcaraz** [reply]
Hello Vadim,

I am not sure what happened, maybe I added two times the same file and it merged them, but I ran it again and now it worked fine. 
Thank you for your help! 

Best,
Paula
